.github/workflows/SDSGAS-CI.yaml

```diff
@@ -0,0 +1,25 @@
+name: SDSGAS-CI
+on:
+  push:
+    branches: main
+  pull_request:
+    branches: main
+jobs:
+  build:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          path: main
+      - name: Use Node.js v18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Install Dependencies
+        run: npm ci
+      - name: Run linters
+        run: npm run verify-forma
```

Comment:
Should this be `verify-format`?

Reply:
Yes it should, fixed on most recent commit